metatron-app/metatron-discovery
# #1921 The number of data sources in the workspace UIs are different

### Description

Adjust the data source list screen due to different numbers in the workspace and data source list screens.

**Related Issue** : #1921 


### How Has This Been Tested?

1. Go to shared workspace
2. Check if the screen has been changed according to the link below.
    https://github.com/metatron-app/metatron-discovery/issues/1921#issuecomment-487852149
    https://github.com/metatron-app/metatron-discovery/issues/1921#issuecomment-488579939

#### Need additional checks?
Also make sure that the data source selection screen on the dashboard creation screen works without any problems.

### Types of changes



### Checklist:


### Additional Context
